joepegs.collections.json

```diff
@@ -6,28 +6,206 @@
+        "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/collections-list/main/images/0x4245a1bD84eB5f3EBc115c2Edf57E50667F98b0b/logo.png",
+        "name": "Hoppers Game",
+        "symbol": "HOP",
+        "twitterURI": "https://twitter.com/HoppersGame",
+        "verified": "verified",
+        "websiteURI": "https://hoppersgame.io/"
+        
+        
+    },
+    
+      "0xa7abc3736bc5a4d3d4730eac5a1e487e56c86789": {
+        "bannerURI": "https://raw.githubusercontent.com/traderjoe-xyz/collections-list/main/images/0xa7abc3736bc5a4d3d4730eac5a1e487e56c86789/banner.png",
+        "description": " Fırat keeps reigning as the most famous comics hero of Turkey since he came into our lives in 2007. The cartoonish life of this kid, who has a language and a world-view of his own, has glimpses of everybody's childhood one way or the other. Fırat has an answer to every situation he finds himself in. He suffers no traumas whatsoever. He never cries, even if he falls. He finds joy in falling in a cool manner. He can imagine block-buster features with rockets exploding and cars crashing one another with a slipper or a triple socket. His only concern is to grow up fast, when he finally does, he imagines he will be the coolest person on the planet. Fırat has 5 published books up to now, each of them a best seller. He now multiplies in NFT.", 
+        "discordURI": "https://discord.gg/firat",
+        "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/collections-list/main/images/0xa7abc3736bc5a4d3d4730eac5a1e487e56c86789/logo.png",
+        "name": "Firat",
+        "symbol": "FIRAT_NFT",
+        "twitterURI": "https://twitter.com/firatinsayfasi",
+        "verified": "verified_trusted",
+        "websiteURI": "www.firatnft.com"
+        
+    },
+    {
+      "0x6a2b2e8123a046e643faf2cb78d86eb4e8496d2f": {
+        "bannerURI": "https://raw.githubusercontent.com/traderjoe-xyz/collections-list/main/images/0x6a2b2e8123a046e643faf2cb78d86eb4e8496d2f/banner.png",
+        "description": "In 2077 Alfa created 8,888 cloned versions of her consciousness. They were each placed into a robotic suit that the conscious could control. In essence, each Warrior is Alfa. They work to her regime, order and demands. Their mission is to defend Alfa at all costs and to further the knowledge and evolution of the human species. Alfa has stated that only 8,888 AVAXWARRIORS will be created. They will all be unique 1 of 1's.", 
+        "discordURI": "discord.gg/avaxwarriors",
+        "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/collections-list/main/images/0x6a2b2e8123a046e643faf2cb78d86eb4e8496d2f/logo.png",
+        "name": "AVAXWARRIORS",
+        "symbol": "AVAXWARRIORS",
+        "twitterURI": "https://twitter.com/AVAXWARRIORS",
+        "verified": "verified",
+        "websiteURI": "https://avaxwarriors.io/home"
+        
+        
+    },
+        
+      "0xf675a87397a6239eaf95ad948670a5b19d076c59": {
```

Comment:
think the spacing might be off here, you can paste this into https://jsonformatter.org/json-pretty-print and get it formatted